charts/v2.1/cray-hms-bss/templates/tests/test-functional.yaml

```diff
@@ -30,4 +56,4 @@ spec:
           image: "{{ .Values.tests.image.repository }}:{{ .Values.global.testVersion }}"
           imagePullPolicy: "{{ .Values.tests.image.pullPolicy }}"
           command: ["/bin/sh", "-c"]
-          args: [ "until curl --head localhost:15000 ; do echo Waiting for Sidecar; sleep 3 ; done ; echo Sidecar available; entrypoint.sh functional -c /src/libs/tavern_global_config.yaml -p /src/app"]
+          args: [ "entrypoint.sh functional -c /src/libs/tavern_global_config.yaml -p /src/app"]
```

Comment:
Since istio 1.7, the same functionality is achieved with configuration option `holdApplicationUntilProxyStarts`, provided through pod annotation.